includes/wikia/helpers/CrossOriginResourceSharingHeaderHelper.php

```diff
@@ -50,11 +61,17 @@ public function setHeaders( WikiaResponse $response, $mergeExisting = true ) {
 					$response->removeHeader( $headerName );
 
 					foreach ( $headers as $header ) {
-						$valuesToSet = array_merge( $valuesToSet, explode( self::HEADER_DELIMETER, $header['value'] ) );
+						if ( strpos( $header['value'], self::HEADER_DELIMETER ) !== false ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to not nest more than 4 "if", "for", "while", "switch" and "try" statements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS134)